astropy/wcs/src/wcslib_wrap.c

```diff
@@ -3993,12 +3994,32 @@ PyWcsprm_set_zsource(
   return set_double("zsource", value, &self->x.zsource);
 }
 
+
+static PyObject*
+PyWcsprm_get_aux(
+    PyWcsprm* self,
+    /*@unused@*/ void* closure) {
+
+  PyObject* result;
+  double test = 1.2;
+
+  result = (PyObject *)PyAuxprm_cnew((PyObject *)self, self->x.aux);
+
+  if (result == NULL) {
+    Py_DECREF(result);
```

Comment:
I do not think you need to decrement `result` when it is `NULL`.

Reply:
Fixed